README.md

```diff
@@ -33,11 +33,12 @@ This is a quick installation for use with the cloud environment https://c9.io -
 
 1. If you have a GitHub account, visit https://c9.io and log in with the GitHub button.
 2. Fork this repository to your own GitHub account, creating a `yourname/plots2` project.
-3. Name your project, then enter `yourname/plots2` in the "Clone from Git or Mercurial URL" field, and press **Create Workspace** 
+3. Name your project, then enter `yournameplots2` in the "Clone from Git or Mercurial URL" field, and press **Create Workspace** 
```

Comment:
Hi, just reading through the changes carefully, "yourname/plots2 does not work with symbols" -- was that in reference to the workspace name? I had meant it to be for the "Git or Mercurial URL", so what about expanding it to: 

```
3. Name your project `plots2` under "Workspace name".
4. Enter `your_github_username/plots2` in the "Clone from Git or Mercurial URL" field, and press **Create Workspace** (don't choose a template!)
```